src/modules/formBuilder/components/itemEditor/FormEditorItemProperties.tsx

```diff
@@ -289,12 +289,26 @@ const FormEditorItemProperties: React.FC<FormEditorItemPropertiesProps> = ({
                 {initialItem.linkId}
               </CommonLabeledTextBlock>
             )}
-            <ControlledTextInput
-              control={control}
-              // TODO(#5776)
-              name='mapping.customFieldKey'
-              label='Mapping for custom field key'
-            />
+            {/* If this is a new item and it's a question, allow user to choose CustomDataElementDefinition key from existing keys. If left empty, a custom_field_key and CDED will be generated when the form is published */}
```

Comment:
Leaving this out for now since the pick list is not supported yet. Will be addressed with https://github.com/open-path/Green-River/issues/5776